content/docs/user-guide/contributing/blog.md

```diff
@@ -41,7 +41,6 @@ tags:
   - Version Control
   - AI
 ---
-
 ```
```

Comment:
Shoot so now Restyled removed this line but `prettier` adds it back, so the check is failing... Looks like there's still a config discrepancy somewhere 😕

Reply:
Hmmm I reintroduced the line in a9b208e6 and Restyled left it alone this time 🤷